robocop/rules.py

```diff
@@ -137,3 +137,14 @@ def __lt__(self, other):
 
     def get_fullname(self):
         return f"{self.severity.value}{self.rule_id} ({self.name})"
+
+    def to_json(self):
+        return {
+                "source": self.source,
+                "line": self.line,
+                "col": self.col,
+                "severity": self.severity.value,
+                "rule_id": self.rule_id,
+                "desc": self.desc,
```

Comment:
Hmm, why not to put here the whole word (`description`)?

Reply:
you mean something like `"description": self.desc,` right?

Reply:
Not Mateusz but I will answer: yes exactly. If you want to update it you will need also to update tests (under tests / utests/test_reports.py)

Reply:
Yes. Same goes with the "col" key. I think we don't need to save space by reducing the names. Would be more clear for everyone that "desc" is "description" and "col" is "column" IMO.